src/03-modality-agnostic-files.md

```diff
@@ -251,6 +251,71 @@ to date of birth.
 }
 ```
 
+## Samples file
+
+Template:
+
+```Text
+samples.tsv
+samples.json
+```
+
+The purpose of this RECOMMENDED file is to describe properties of samples
```

Comment:
should this be recommended or required if you have a sample entity. i don't think i would want people to use the sample entity to run disambiguation routes, and we would want to know the type of sample.

Reply:
I addressed this in another similar question [here](https://github.com/bids-standard/bids-specification/pull/812#discussion_r646894795).